cms-2016-pileup-dataset/inputs/recid_info.py

```diff
@@ -0,0 +1,3 @@
+RECID_INFO = {
+    "/Neutrino_E-10_gun/RunIISummer20ULPrePremix-UL16_106X_mcRun2_asymptotic_v13-v1/PREMIX": 30566,
```

Comment:
Note that we will have to change the record ID due to having extra RECO configuration files for collision data. I think 30595 should be free.

Reply:
DOI could be 10.7483/OPENDATA.CMS.VWUA.G7SB